source/docs/software/roborio-info/roborio-introduction.rst

```diff
@@ -0,0 +1,13 @@
+roboRIO Introduction
+====================
+
+.. image:: images/roborio-introduction/roborio.png
+  :width: 500
+
+The roboRIO is designed specifically with FIRST in mind. The roboRIO has a basic architecture of a Real-Time processors + FPGA (field programmable gate array) but is more powerful, lighter, and smaller than some similar systems used in industry.
```

Comment:
This isn't actually correct. The roboRIO is not specifically designed for FIRST.

Reply:
Yes it was, but it certainly was designed "specifically with _FIRST_ in mind" which is slightly broader than "designed for _FIRST_"

Reply:
My apologies, I thought it was designed originally for industry usage.

Reply:
Lineage is cRIO (designed for industry) -> myRIO (designed for education) -> roboRIO (designed for FRC)

Reply:
Interesting.